tools/test-popular-containers/build_rootfs.sh

```diff
@@ -43,7 +43,7 @@ function make_rootfs {
     umount -l mnt
     rmdir mnt
 
-    systemd-nspawn --pipe -i $IMG /bin/sh <<EOF
+    systemd-nspawn --timezone=off --pipe -i $IMG /bin/sh <<EOF
```

Comment:
what's the motivation for the timezone argument?

Reply:
Building the rootfs failed for 24.04 without it, with a conflict that it could not replace it (probably because systemd-nspawn bind mounts it).

Reply:
I would mention that in the commit

Reply:
Done

Reply:
nit: Could we add it as a code comment instead/as well?

Reply:
Done